encodings/lp_file.lp

```diff
@@ -0,0 +1,3 @@
+variable_type("x",int,range,(value,2,value,3)).
```

Comment:
do we need this file ?
lp_file.lp is a funny name

Reply:
I changed both tmp.lp and lp_file to out.lp